packages/ui/src/modules/onboarding/views/main.js

```diff
@@ -16,8 +16,9 @@ import Skip from './skip.js';
 import OutroSkip from './outro-skip.js';
 import OutroSuccess from './outro-success.js';
 
-const TERMS_AND_CONDITIONS_URL =
-  'https://www.ghostery.com/privacy/ghostery-terms-and-conditions?utm_source=gbe';
+const TERMS_AND_CONDITIONS_URL = `https://www.${
+  chrome.runtime.getManifest().debug ? 'ghosterystage' : 'ghostery'
```

Comment:
@smalluban this patter repeats. Do you think it is worth to encapsulate this it?